core/src/components/alert/alert.tsx

```diff
@@ -1,6 +1,7 @@
 import type { ComponentInterface, EventEmitter } from '@stencil/core';
 import { Component, Element, Event, Host, Listen, Method, Prop, Watch, forceUpdate, h } from '@stencil/core';
 
+import { config } from '../../global/config';
```

Comment:
Bikeshedding config names:

- `enableInnerHTMLParsing`
- `enableCustomHTMLParsing`
- `enableHTMLParsing`
- `enableCustomHTMLContent`

Reply:
Maybe something more specific like `enableHTMLInPropValues`?

Reply:
I lean towards naming that hints at `innerHTML`.

e.g.: `enableInnerHTMLParsing`, `enableUseInnerHTML`, `useInnerHTML`, etc.

While this does deal with parsing/sanitization/etc., the primary function is deciding if we render as a string or bind to `innerHTML`.

Reply:
I like `enableInnerHTMLTemplates` and +1 to being specific about custom templates.

Reply:
Ok one more thought 😂  We have `swipeBackEnabled` and `sanitizerEnabled` in the config. Should we name this `innerHTMLTemplatesEnabled` instead?

Reply:
I'm confused on what `swipeBackEnabled` has to do with this. I thought it was for swipe gestures? 

For the `sanitizerEnabled` - this just enables the sanitizer itself, so if this is `true` and `enableInnerHTMLTemplates` is false, then it is technically not going to be used. Maybe we should document the relationship here? I imagine `sanitizerEnabled` is useful for people who want to go crazy passing things into the `innerHTML` without us stripping anything out. 

I don't think we should rename `sanitizerEnabled` to `innerHTMLTemplatesEnabled` since it doesn't seem like that's what it is actually doing, but correct me if I am wrong!

Reply:
I brought those up as examples of the naming style in this config. We have `swipeBackEnabled` and `sanitizerEnabled` configs, but this PR proposes `enableInnerHTMLTemplates` which breaks the `[name]Enabled` format we've been using. I was not proposing we change `swipeBackEnabled` or `sanitizerEnabled`.

Reply:
Oh I see, sorry I was so confused. 😂 

Yes, I think we should keep it consistent with the other ones. I am good with `innerHTMLTemplatesEnabled`.

Reply:
Updated in https://github.com/ionic-team/ionic-framework/pull/26956/commits/430e6b96f23f076aad20ee6eca1d926b6898585a